src/api/commands/currencyprice/CurrencyPriceAddCommand.ts

```diff
@@ -0,0 +1,66 @@
+import { inject, named } from 'inversify';
+import { validate, request } from '../../../core/api/Validate';
+import { Logger as LoggerType } from '../../../core/Logger';
+import { Types, Core, Targets } from '../../../constants';
+import { CurrencyPriceService } from '../../services/CurrencyPriceService';
+import { RpcRequest } from '../../requests/RpcRequest';
+import { RpcCommandInterface } from '../RpcCommandInterface';
+import { Commands } from '../CommandEnumType';
+import { BaseCommand } from '../BaseCommand';
+import { MessageException } from '../../exceptions/MessageException';
+import { CurrencyPriceParams } from '../../requests/CurrencyPriceParams';
+import { SupportedCurrencies } from '../../enums/SupportedCurrencies';
+
+export class CurrencyPriceAddCommand extends BaseCommand implements RpcCommandInterface<any> {
+
+    public log: LoggerType;
+
+    constructor(
+        @inject(Types.Core) @named(Core.Logger) public Logger: typeof LoggerType,
+        @inject(Types.Service) @named(Targets.Service.CurrencyPriceService) private currencyPriceService: CurrencyPriceService
+    ) {
+        super(Commands.CURRENCYPRICE_ADD);
+        this.log = new Logger(__filename);
+    }
+
+    /**
+     *
+     * @param data
+     * @returns {Promise<Bookshelf.Model<any>>}
+     */
+    @validate()
```

Comment:
describe the params  

  /**
     *
     * data.params[]:
     *  [0]: fromCurrency
     *  [1]: toCurrency
     *  [...]: toCurrency
     *
     * description ...
     *
     * ...
     */